keylime/cloud_verifier_common.py

```diff
@@ -206,7 +209,11 @@ def process_quote_response(agent, json_response):
                                      agent['tpm_policy'],
                                      ima_measurement_list,
                                      agent['allowlist'],
+                                     mb_measurement_list,
+                                     #agent['mb_intended_state'],
```

Comment:
You probably want to remove this (and other commented code below). Otherwise rest LGTM.